menuinst/platforms/win.py

```diff
@@ -64,6 +65,22 @@ def quick_launch_location(self) -> Path:
     def desktop_location(self) -> Path:
         return Path(windows_folder_path(self.mode, False, "desktop"))
 
+    @property
+    def terminal_profile_location(self) -> Path:
+        if self.mode == "system":
+            warnings.warn("Terminal profiles are not available for system level installs")
+            return None
+        profile_path = (
+            Path(windows_folder_path(self.mode, False, "localappdata"))
+            / "Packages"
+            / "Microsoft.WindowsTerminal_8wekyb3d8bbwe"
```

Comment:
Is this the actual folder name? That random string thing looks autogenerated but maybe is not?

Reply:
I agree that it looks that way, but it is not auto-generated: https://learn.microsoft.com/en-us/windows/terminal/install#settings-json-file

I should point out in the documentation though that it adds to the stable release. Or I expand the functionality to install into all instances (stable, preview, and unpackaged) it finds. What do you think?

Reply:
Stable is enough, I'd say. No strong opinion if you want to add more, though.

Reply:
Might as well